docs/immortal-empires/skaven.md

```diff
@@ -0,0 +1,23 @@
+## Throt the Unclean - Clan Moulder
+
+* **Route I - Yes-yes, prepare for war!**
```

Comment:
You changed the tables but forgot to update the markdown, for both route name and rewards. Nice changes tho.

Reply:
Will do myself